settings.gradle.kts

```diff
@@ -21,3 +21,4 @@ dependencyResolutionManagement {
 rootProject.name = "credentials-manager-lib"
 
 includeBuild("build-conventions")
+include(":utils")
```

Comment:
why utils as a name? why not `in-memory`

Reply:
@y9vad9 i think - creating single module for single class not cool... i named it's as `utils` for future features.